packages/react-components/react-storybook/README.md

```diff
@@ -22,8 +22,33 @@ You need to register fluentui decorators on your particular level (global/story/
 // @filename: .storybook/preview.js
 
 import { withKnobs } from '@storybook/addon-knobs';
-import { withStrictMode } from '@fluentui/react-storybook';
+import { withStrictMode, withFluentVrTestVariants } from '@fluentui/react-storybook';
 
 // Register decorators on global level
-export const decorators = [withKnobs, withStrictMode];
+export const decorators = [withKnobs, withStrictMode, withFluentVrTestVariants];
```

Comment:
@ling1726  mentioned good suggestion that this should live as part of `react-storybook-addon` which is the storybook addon package that follows latest storybook API's. 

Thus ideal approach how to register the decorator would be by just simply registering the addon in particular storybook

```js
{
 addons: [
    '@storybook/addon-essentials',


    // internal monorepo custom addons

    /**  @see ../packages/react-components/react-storybook-addon */
    loadWorkspaceAddon('@fluentui/react-storybook-addon'),
  ],
}
```

Reply:
Gotcha - will move this to the react-storybook-addon package with an updated implementation